frontend/src/components/GrantAccess.vue

```diff
@@ -99,22 +97,21 @@ function toRequestPayload(formData: any) {
     return request;
 }
 
-function statusSelected(evt: any) {
-    forestClient.value = null;
-    formData.value.forestClientNumber = null;
-    formData.value.role = JSON.parse(evt.target.value);
+function onRoleSelected(evt: any) {
+    forestClient = null;
+    formData.value.forestClientNumber = "";
```

Comment:
Not sure why using blank forest client number instead of null.

Reply:
This "" or null was also part of trouble for me to resolve due to Yup upgrade to 1.0.0 without changing any code.
The Yup form input schema checks seems more rigid than previous. Since we initially had this as Yup.string() input type for "forestClientNumber" input and checking (various things, like min()), it now complaints it is not string type (which initially I thought is accepted if null... maybe I am mixing with java String wrapper type with typescript, so you will need to do acceptable argument types on typscript for like: string || null || undefined).
Anyway, I am a little bit confused about Yup as well as I am learning, but this is the thing I need to change for Yup for validation to work.